<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditor</class>
 <widget class="QWidget" name="ItemEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="5">
    <widget class="QPushButton" name="itemSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="4">
    <widget class="QFrame" name="frameAction">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QComboBox" name="actionList"/>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="actionDelete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="actionAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelAction">
        <property name="text">
         <string>Action</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QWidget" name="actionWidget" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="8" column="0">
          <widget class="QCheckBox" name="actionHasHookDoors">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="2" colspan="2">
          <widget class="QLineEdit" name="actionHookUse"/>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QCheckBox" name="actionCombat">
           <property name="text">
            <string>Can only be performed during combat</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QCheckBox" name="actionHasHookWeapon">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Doors</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QSpinBox" name="actionDamageMax">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QSpinBox" name="actionApCost">
           <property name="maximum">
            <number>99</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QCheckBox" name="actionTargeted">
           <property name="text">
            <string>Needs to be targeted to something</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Targeted</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QSpinBox" name="actionDamage">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QSpinBox" name="actionRange">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Use</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Combat</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="actionHasHookUse">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Characers</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Action Points</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Script hooks:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2" colspan="2">
          <widget class="QLineEdit" name="actionHookCharacters"/>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="actionHasHookCharacters">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Damage range</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Range</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2" colspan="2">
          <widget class="QLineEdit" name="actionHookDoors"/>
         </item>
         <item row="9" column="2" colspan="2">
          <widget class="QLineEdit" name="actionHookOthers"/>
         </item>
         <item row="9" column="0">
          <widget class="QCheckBox" name="actionHasHookOthers">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>As a weapon</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Others</string>
           </property>
          </widget>
         </item>
         <item row="10" column="2" colspan="2">
          <widget class="QLineEdit" name="actionHookWeapon"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" rowspan="3" colspan="4">
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Icon</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="itemIcon"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>3D Model</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="itemModel"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>3D Texture</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="itemTexture"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>3D Scale</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="itemScale"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Weight</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="itemWeight">
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Visible</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="itemHidden">
        <property name="text">
         <string>Can be seen in the inventory</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Script File</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="itemScript"/>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="itemModeMouth">
        <property name="text">
         <string>Mouth Weild</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="scriptMouthWeild"/>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="itemModeMagic">
        <property name="text">
         <string>Magic Weild</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="scriptMagicWeild"/>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="itemModeBattleSaddle">
        <property name="text">
         <string>BattleSaddle Wild</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="scriptBattleSaddleWeild"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QLabel" name="labelCurrentItem">
     <property name="text">
      <string>Object</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="6">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="itemAdd">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="itemDelete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="itemSearch"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QListWidget" name="itemList"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
